crates/jstzd/tests/jstzd_test.rs

```diff
@@ -164,18 +280,12 @@ async fn read_bootstrap_contracts() -> Vec<BootstrapContract> {
     contracts
 }
 
-async fn check_bootstrap_contracts(octez_client: &OctezClient) {
```

Comment:
You can move this function back to this position from above so that there are fewer lines modified